lte/gateway/python/magma/pipelined/tests/snapshots/test_uplink_bridge.UplinkBridgeWithNonNATTest.testFlowSnapshotMatch.snapshot

```diff
@@ -4,7 +4,6 @@
  priority=100,in_port=2 actions=mod_dl_src:02:bb:5e:36:06:4b,output:3
  priority=101,ip,in_port=3,nw_dst=1.1.11.1 actions=LOCAL
  priority=101,ipv6,in_port=3,ipv6_dst=fe80::48a3:2cff:fe1a:abcd actions=LOCAL
- priority=101,ipv6,in_port=3,ipv6_dst=ff02::1:ff00:0/104 actions=output:2,LOCAL
```

Comment:
Just curious, do you have an explanation why this entry needs to be removed (also in other snapshots)?

Reply:
Initially when I started testing ipv6 data, DL data sent from trfserver vm were not reaching dev vm. So, a piece of code was added in pipelined/app/uplink_bridge.py and this was merged to master. For this a new flow was added in the snapshot. But later found that the fix in pipelined/app/uplink_bridge.py was creating a different problem where UE which was earlier able to send UL packets was not able to connect with trfserver vm(traffic server) at all. So, in my PR I removed the code added in pipelined/app/uplink_bridge.py and fixed the DL data issue by adding a DL route via dev vm in trfserver. That’s the reason the flow has to be deleted now in all the related snapshots

Reply:
Got it, so this PR includes a revert of https://github.com/magma/magma/pull/12165. Thank you